eng/scripts/docs/Docs-ToC.ps1

```diff
@@ -0,0 +1,57 @@
+function Get-NamepspacesFromDll($dllPath) {
+    $file = [System.IO.File]::OpenRead($dllPath)
```

Comment:
The code of parsing metadata out actually sourced from @heaths work. Thanks @heaths !